src/repository/index.ts

```diff
@@ -105,6 +110,7 @@ export default class Repository extends EventEmitter implements EventEmitter {
     bootstrapProvider,
     bootstrapOverride = true,
     storageProvider,
+    eventSource,
```

Comment:
we make EventSource injectable so that it's easy to swap in testing for a simple mock